<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logTable</class>
 <widget class="QWidget" name="logTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>logTable</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="threadIdLabel">
           <property name="text">
            <string>线程号:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="threadIdText"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="levelLabel">
           <property name="text">
            <string>级别:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="levelCombo">
           <item>
            <property name="text">
             <string>all</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>baseddebug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>warn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>error</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fault</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="filiterLabel">
             <property name="text">
              <string>内容:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filiterContext"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="filiterButton">
           <property name="text">
            <string>过滤(&amp;L)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="forwardButton">
           <property name="text">
            <string>前一条(&amp;E)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backwardButton">
           <property name="text">
            <string>后一条(&amp;R)</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>导出(&amp;X)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rollingCheck">
         <property name="text">
          <string>滚动</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTableView" name="table"/>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>日志内容</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>filiterButton</tabstop>
  <tabstop>forwardButton</tabstop>
  <tabstop>backwardButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>threadIdText</tabstop>
  <tabstop>levelCombo</tabstop>
  <tabstop>filiterContext</tabstop>
  <tabstop>table</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>rollingCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
